hardhat.config.js

```diff
@@ -27,6 +27,12 @@ const runScript = (script) => {
   });
 };
 
+task("deploy-argent", "Deploy argent", async () => {
```

Comment:
I'm not sure this is necessary since you can simply execute deploy-argent.js with the command `npx hardhat run deploy-argent.j`.